### PR TITLE
Update: Added runtime details for wf swarm

### PR DESCRIPTION
**Pull request by rhoads-zach**
@heiko-braun and @inoxx03 let me know what you think.

### CODE REVIEW

File: docs/topics/wf-swarm-runtime-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+= Runtime Details
+//http://wildfly-swarm.io
+
+{WildFlySwarm} deconstructs the features in link:http://wildfly.org[WildFly Java Application Server] and allows them to be selectively reconstructed based on the needs of your application. This allows you to create microservices that run on a _just-enough-appserver_ that supports the exact subset of JavaEE APIs you need. Check out xref:wf-swarm-additional-resources[Additional Resources] for further reading on {WildFlySwarm}.
```
Comment (heiko-braun) [inline docs/topics/wf-swarm-runtime-overview.adoc]: Turn `JavaEE APIs` into API's.

**Comment by heiko-braun** [comment]
@kenfinnigan You might want to take a look too

File: docs/topics/wf-swarm-runtime-overview.adoc (hunk 1 above)
Comment (Ladicek) [inline docs/topics/wf-swarm-runtime-overview.adoc]: "WildFly Java Application Server" should probably read "WildFly Java EE Application Server"?

Comment (Ladicek) [inline docs/topics/wf-swarm-runtime-overview.adoc]: I'd say "the exact subset of JavaEE APIs you need" should actually read "the exact subset of Java EE APIs you need". I.e., space between "Java EE", but "APIs" looks OK to me.

**Comment by Ladicek** [comment]
Otherwise LGTM.

File: docs/topics/wf-swarm-runtime-overview.adoc (hunk 1 above)
Comment (heiko-braun) [inline docs/topics/wf-swarm-runtime-overview.adoc]: We need to detach Swarm from the Term Java EE.

**Comment by rhoads-zach** [comment]
Thank you for the feedback @Ladicek and @heiko-braun, I've updated per the feedback and removed references to Java EE.

Let me know what you think.

File: docs/topics/wf-swarm-dev-guide-additional-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[[wf-swarm-additional-resources]]
+= Additional Resources
+
+* link:https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/v/2017.6.0/index.html[{WildFlySwarm} User Guide]
```
Comment (Ladicek) [inline docs/topics/wf-swarm-dev-guide-additional-resources.adoc]: We should probably also mention https://howto.wildfly-swarm.io/ and https://reference.wildfly-swarm.io/. My understanding is that the User Guide is being slowly phased out.

**Comment by Ladicek** [comment]
Otherwise LGTM.

**Comment by inoxx03** [comment]
merged: cdd7f05ddfbdee11ff5e1578800c430be1cb29a8

**Pull request closed without merging**